src/bsoncxx/document/value.cpp

```diff
@@ -53,6 +53,9 @@ value& value::operator=(const value& rhs) {
     return *this;
 }
 
+value::value(value&& rhs) noexcept: _data{std::move(rhs._data)}, _length{std::move(rhs._length)} {
```

Comment:
No, the other versions passed: https://evergreen.mongodb.com/version/569577b43ff12266560005d0_0

(except for OSX, since I hadn't rebased to get the change lowering the compiler minimum)